CONTRIBUTING.md

```diff
@@ -115,28 +144,231 @@ our project's repositories. The [Developer Certificate of Origin
+
+1. You can create a draft or WIP pull request at any time. You may also
+ mark the PR as not ready by assigning the label `do-not-merge/wip` to it.
+ Reviewers will ignore it mostly unless you mention someone and ask for help.
+ Feel free to open one and use the pull request to see if the CI passes.
+ Once you are ready for a review, remove the WIP or click "Ready for Review" and
+ leave a comment that it's ready for review.
+ Give the PR a descriptive title, that would be appropriate as a commit message once
+ your work is merged. Include additional information, as appropriate, in the PR
+ description paragraph to explain what you have done.
+ If your PR fixes any issues, the description should refer to it (e.g., "fixes #NNN") or
+ you can associate it using "linked issues". Either of these links your PR to the issue
+ and automatically closes the issue when the PR is accepted.
+ Do **not** use the PR comments to document the code as they will most likely never be
+ seen after the PR is merged. Please **do** make your code self-explanatory. If there is
+ information that is needed by a programmer reading the source code, you should put it in
+ a code comment. This also applies to answering questions from reviewers: it is best done
+ by clarifying the code or add documentation, rather than in the PR comment thread.
+
+1. A reviewer will assign themselves to the pull request. If you don't see anyone
+ assigned after 3 business days, you can leave a comment asking for a review, or ping in
+ slack. Sometimes we have busy days, sick days, weekends and vacations, so a little
+ patience is appreciated! 🙇‍♀️
+
+1. The reviewer will leave feedback.
+
+* `nits`: These are suggestions that you may decide to incorporate into your pull
+ request or not without further comment.
+* Requests for change in the PR contents. These require resolution before the PR is
+ merged.
+* It is okay to clarify if you are being told to make a change or if it is a suggestion.
+
+If you agree with a code review comment and do what it suggests, don't respond in the
+ GitHub code review system. Simply resolve the conversation. Respond if more discussion
+ is needed, such as asking a follow-up question or explaining why you disagree with the
+ suggestion. If the reviewer is asking a question, then usually the best way to answer
+ it is by improving the code or documentation. Answering it only in the code review
+ will not help future programmers after the PR is merged.
+
+Respond to the feedback by making changes in your working copy, committing them, and
+ periodically pushing them to GitHub when the tests pass locally. As soon as you
+ receive feedback, you can start working on it. The reviewer should assign the code
+ review back to you, but they might forget, so don't wait for that.
+
+After you have made the changes (in new commits please!), leave a comment asking the
+ the reviewer to take another look. If 3 business days go by with no review, it is okay
```

Comment:
ditto, too formal :)

Reply:
As a contributor I would like to know if it is ok to nudge the maintainers or not and if so, when?
But maybe that's just me ;-)